database/seeders/DefaultSkills.php

```diff
@@ -29,7 +86,42 @@ public function run(): void
             ['skill_name' => 'Electronics safety', 'category' => 2],
             ['skill_name' => 'Replacing screens', 'category' => 2],
         ];
+    }
+
+    /**
+     * Seed skills to database based on configuration
+     */
+    private function seedSkills(array $data): void
+    {
+        $truncate = env('SEEDING_TRUNCATE_SKILLS', true);
+
+        if ($truncate) {
+            Log::info('Truncating and re-inserting all skills');
+            DB::table('skills')->truncate();
+            DB::table('skills')->insert($data);
+            Log::info('Inserted ' . count($data) . ' skills');
+            return;
+        }
+
+        Log::info('Merging new skills only');
+        $added = 0;
+        $modified = 0;
+
+        foreach ($data as $entry) {
+            $skill = DB::table('skills')->where('skill_name', $entry['skill_name'])->first();
+
+            if (is_null($skill)) {
+                DB::table('skills')->insert($entry);
+                $added++;
+                continue;
+            }
+
+            if (empty($skill->description) && !empty($entry['description'])) {
+                DB::table('skills')->where('id', $skill->id)->update(['description' => $entry['description']]);
```

Comment:
If we switch to `Skill::firstWhere`, we can do something like this which will read easier:

```
$skill->description = $entry['description'];
$skill->save()
```